java/example-graphsdk-client/src/test/kotlin/io/amplica/graphsdk/LibraryTest.kts

```diff
@@ -31,6 +31,16 @@ class LibraryTest {
         graph.finalize()
     }
 
+    @Test
```

Comment:
We should remove this. This tries to test against the uploaded jar but since the uploaded jar does not have `getTestnetPaseo()` it is failing.